src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationMercuryTests.java

```diff
@@ -53,7 +53,8 @@ public void mercury_navigation_navigationOnNonEnglishWiki() {
     );
   }
 
-  @Test(groups = "mercury_navigation_navigationElementsUserLoggedIn")
+  // TODO: enable as a part of fix in IRIS-4209
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)